### ISSUE TITLE
Optimized Loss Value?

### ISSUE THREAD

**Issue by GuanzhaoLi** [qqwweee/keras-yolo3#181]
Hi Guys!
I'm trying to train the model with my own customized object(class=1) for 1000 epochs and got loss=110 something. I'm wondering what is the optimized value that we finally want to achieve for loss value?

Thank you all in advance!

**Comment by Alphonses** [comment]
ok，1000 epochs is extremely large! ! you should  think about "is label  extremely  great?","is dataset large enough?",what's more,you can set " strategy of learn-rate"

**Comment by eain3314** [comment]
你好  你的损失降下来了吗

**Comment by Alphonses** [comment]
降不下来，我换了pytorch版的，效果不错！



------------------ 原始邮件 ------------------
发件人: "eain3314"<notifications@github.com>; 
发送时间: 2019年5月28日(星期二) 下午5:31
收件人: "qqwweee/keras-yolo3"<keras-yolo3@noreply.github.com>; 
抄送: "Alphonse"<1657580398@qq.com>; "Comment"<comment@noreply.github.com>; 
主题: Re: [qqwweee/keras-yolo3] Optimized Loss Value? (#181)




你好  你的损失降下来了吗
